app/controllers/idv/welcome_controller.rb

```diff
@@ -68,5 +71,25 @@ def cancel_previous_in_person_enrollments
         current_user,
       )
     end
+
+    def passport_allowed?
+      return if resolved_authn_context_result.facial_match?
+      return if doc_auth_vendor == Idp::Constants::Vendors::SOCURE
```

Comment:
I prefer predicates to return a boolean, not `nil`

```suggestion
      return false if resolved_authn_context_result.facial_match?
      return false if doc_auth_vendor == Idp::Constants::Vendors::SOCURE
```